packages/studio-be/src/core/bots/bot-service.ts

```diff
@@ -63,7 +63,9 @@ const BotCreationSchema = Joi.object().keys({
       id: Joi.string()
     }
   },
-  locked: Joi.bool()
+  locked: Joi.bool(),
+  isCloudBot: Joi.bool().optional(),
+  bpCloudApiKey: Joi.string().optional()
```

Comment:
yo @rob-gordon 

what are those fields? I've `ctrl+shift+f` in the studio codebase (on branch`next`) and I can't seem to find any usage... 

Is this PR a WIP ? If so, please make it a draft, so we don't review it too soon

Reply:
@franklevasseur Apologies! I had this open but hadn't added a reviewer thinking no one would look at it. I'll work with drafts from now on 👍🏻 

These fields are needed for https://github.com/botpress/botpress/pull/5704

I pushed one more commit because the [ticket scope changed](https://linear.app/botpress/issue/DEV-2026#comment-d8c75834) this morning, but the PR is really ready for review now. 

Thanks!

Reply:
Alright, no prob!

It's not a thing yet, but since the codebase is all split up, we should start linking PR's of other repo in the description, like:

"This PR goes in hand with this one on the main repo ..."